gui/app/directives/modals/misc/editUrlAllowlistModal.js

```diff
@@ -0,0 +1,180 @@
+"use strict";
+
+(function() {
+
+    const fs = require("fs");
+
+    angular.module("firebotApp")
+        .component("editUrlAllowlistModal", {
+            template: `
+            <div class="modal-header">
+                <button type="button" class="close" ng-click="$ctrl.dismiss()"><span>&times;</span></button>
+                <h4 class="modal-title">Edit URL Allowlist</h4>
+            </div>
+            <div class="modal-body">
+                <p class="muted" style="margin-bottom:10px;">URLs found in messages containing any words or phrases listed here will be automatically allowed.</p>
+                <p class="muted" style="margin-bottom:20px;">NOTE: If multiple URLs are found in a message and ANY of them are not allowed, the entire message will be deleted.</p>
+                <div style="margin: 0 0 25px;display: flex;flex-direction: row;">
+
+                    <div class="dropdown">
+                        <button class="btn btn-primary dropdown-toggle" type="button" id="add-options" data-toggle="dropdown" aria-haspopup="true" aria-expanded="true">
+                            <span class="dropdown-text"><i class="fas fa-plus-circle"></i> Add Word(s)</span>
```

Comment:
In this context I think this should be updated to `Add URL(s)`

Reply:
Fixed